reconcile/utils/openshift_resource.py

```diff
@@ -464,8 +465,12 @@ def __iter__(self):
                     data = self._clusters[cluster][namespace][resource_type]
                     yield (cluster, namespace, resource_type, data)
 
-    def register_error(self):
+    def register_error(self, cluster=None):
         self._error_registered = True
+        if cluster is not None:
+            self._error_registered_clusters[cluster] = True
 
-    def has_error_registered(self):
+    def has_error_registered(self, cluster=None):
+        if cluster is not None:
+            return self._error_registered_clusters.get(cluster) or False
```

Comment:
`return self._error_registered_clusters.get(cluster, False)`

Reply:
this is not graphql
this is not graphql
this is not graphql

Reply:
thanks!